### ISSUE TITLE
Hyperparameter Sweep

### ISSUE THREAD

**Issue by EricChen424** [bcgsc/GapPredict#145]
Seed Length range

26-26
26-52
26-104
26-max
52-52
52-104
52-max
104-104
104-max

RNN Dim = 512
Embedding Dim = 128
Batch Size = 128
If needed can repeat this for another hyperparameter
10 replicates

Track both training loss, training accuracy, validation loss, validation accuracy at the final model after early stopping; don't care about the model itself
